File: api/src/models/EventosEspecialesModel.js
/*
    Esquema de para la colección EventosEspeciales en la base de datos
    En los esquemas deben poner todos los atributos de la base de datos para poder manipularlos.
*/

const mongoose = require('mongoose');
const EventosEspecialesSchema = new mongoose.Schema({

    _id: { type: Number},
    id_local: { type: Number},
    nombre_evento: { type: String},
    descripcion: { type: String },
    fecha: { type: String },
    hora: { type: String }
},{ collection: 'EventosEspeciales' });

module.exports = mongoose.model('EventosEspeciales', EventosEspecialesSchema)

File: web/MongoProyectoWeb/MongoProyectoWeb/Models/EventosEspecialesModel.cs
using static System.Runtime.InteropServices.JavaScript.JSType;

namespace MongoProyectoWeb.Models
{
    public class EventosEspecialesModel
    {
        public int? _id { get; set; }
        public int? id_local { get; set; }
        public string? nombre_evento { get; set; }
        public string? descripcion { get; set; }
        public string? fecha { get; set; }
        public string? hora { get; set; }
    }
}

